core/ts/index.ts

```diff
@@ -19,5 +21,7 @@ export {
     Poll,
     genProcessVkSig,
     genTallyVkSig,
-    genSubsidyVkSig
+    genSubsidyVkSig,
+    genNewKeyGenerationVkSig,
```

Comment:
Where are these two methods used? Do we need a separate task to implement their usage in the same way other methods here are used? Mainly in the setVerifyingKeys.ts -> setVerifyingKeys method

Reply:
Aha, now i see they are used in the contract e2e test. We still should add them to the method mentione above, lets do a separate task for that. I ll open it.

Reply:
Yes, we must add them to that method in MaciState. Great catch and thank you!

Reply:
https://github.com/0x3327/maci/pull/33/commits/02ed9d98c406f54353634b46f5ad41fcb57a194f